stac_geoparquet/arrow/_to_parquet.py

```diff
@@ -19,69 +19,94 @@ def parse_stac_ndjson_to_parquet(
     chunk_size: int = 65536,
     schema: pa.Schema | InferredSchema | None = None,
     limit: int | None = None,
+    schema_version: Literal["1.0.0", "1.1.0"] = "1.0.0",
```

Comment:
Since we repeat this literal, might make sense to pull it to a constant/typedef, e.g. `SUPPORTED_SCHEMA_VERSIONS` and `DEFAULT_SCHEMA_VERSION`.

Reply:
Agreed. And we'll want to export that somewhere in the public API.